e2e/cypress/tests/integration/enterprise/cloud/billing/notify_admin_spec.js

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Move to utils
+function simulateSubscription(subscription, withLimits = {}) {
```

Comment:
Hey @yasserfaraazkhan , added this e2e tests. Let me know how we can improve them